libqtile/widget/statusnotifier.py

```diff
@@ -94,7 +94,24 @@ def __eq__(self, other):
 
     async def start(self):
         # Create a proxy object connecting for the item.
-        introspection = await self.bus.introspect(self.service, self.path)
+        # Some apps provide the incorrect path to the StatusNotifier object
+        # We can try falling back to the default if that fails.
+        # See: https://github.com/qtile/qtile/issues/3418
+        # Note: this loop will run a maximum of two times and returns False
+        # if the no object is available.
+        found_path = False
+
+        while not found_path:
+            try:
+                introspection = await self.bus.introspect(self.service, self.path)
+                found_path = True
+            except InvalidObjectPathError:
+                logger.info(f"Cannot find {self.path} path on {self.service}.")
```

Comment:
Just because `info` is not enabled by default :)
```suggestion
                logger.info("Cannot find {} path on {}.", self.path, self.service)
```